config/channel/distributed/300-kafka-secret.yaml

```diff
@@ -16,9 +16,9 @@ apiVersion: v1
 data:
   # ALL VALUES NEED TO BE REPLACED
   brokers: RU1QVFk=
```

Comment:
If we're using empty strings now can we update the comment above to say... "# Specify Kafka Broker & Optional Credentials"  or something ?

Reply:
Yes, in fact there's no reason to have that "RU1QVFk=" string in there either; it's confusing for no purpose and it's just as easy to replace an empty brokers string in the e2e-tests.sh script.